ironfish/src/rpc/routes/wallet/create.ts

```diff
@@ -1,31 +1,17 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-import * as yup from 'yup'
-import { ERROR_CODES, ValidationError } from '../../adapters'
-import { ApiNamespace, routes } from '../router'
-
-export type CreateAccountRequest = { name: string; default?: boolean }
-export type CreateAccountResponse = {
-  name: string
-  publicAddress: string
-  isDefaultAccount: boolean
-}
 
-export const CreateAccountRequestSchema: yup.ObjectSchema<CreateAccountRequest> = yup
-  .object({
-    name: yup.string().defined(),
-    default: yup.boolean().optional(),
-  })
-  .defined()
+/**
+ * NOTE: This endpoint will be deprecated in favor of `POST /wallet/createAccount` because
+ * this endpoint does not follow the convention that all of our endpoints should follow which
+ * is the verbObject naming convention, where the verb is the HTTP verb and the object is the
+ * object being acted upon. For example, `POST /wallet/burnAsset` burns an asset.
+ */
```

Comment:
nit - do we want to use the HTTP verbs in documentation since our endpoints aren't necessarily used over HTTP?

Reply:
Done, changed it to remove the http language